docs/blueprints/EP000.rst

```diff
@@ -0,0 +1,156 @@
+
+#########
+Audience
+#########
+The primary audience for EPs are the core developers, as well as network engineers involved at the Kytos Project. However, other parts of the Kytos community may use the EP's to understand many aspects of the project.
+
+#######
+Rationale
+#######
+This meta blueprint is intended to make the contributors workflow more clear about how to write a new blueprint.
+
+#############
+Specification
+#############
+
+Workflow
+**************
+Every blueprint should be follow the steps below in order to be finished:
+ 
+Outline the subject
+=================
+The process to write a blueprint should begins describing a new idea, a specification to something that it was defined to be deployed. It is recommended that a single blueprint contain a single proposal. The more focused the blueprint the more successful it tends to be. The Kytos team have the right to reject or aprove any proposal. If in doubt, create a issue in the Kytos Project at Github and add a coment asking if the the issue could be a blueprint.
+
+Submitting a blueprint
+====================
+After create a first blueprint adding every necessary section (check the sections at "Sections that a blueprint should have") you are encouraged to create a pull request at repository kytos attending the following criteria:
+
+    - The file containing the proposal must be named "EPX.rst";
+
+    - The file must containing a header following the template avaible in Blueprint Header section;
+
+    - The file must int restructured text (RST) format like the others blueprints (as you can see at 'kytos/docs/blueprints').
+
+
+Blueprint review
+===============
+
+The following days after you submit your pull request the Kytos team will be reviewing the document adding comments and suggestions. So is very important that the author of the pull request pay attention to the Kytos team feedback to make review a quick process.
+
+
+Sections that a blueprint should have
+*****************************************************
+
+Header
+=======
+This section must  be use the template available at Blueprint Header section in this blueprint.
+
+Abstract
+========
+A short (~200 word) description of the technical issue being addressed.
+
+Motivation
+=========
+It should clearly explain why the blueprint is being proposed describing any techinal issue that is intended to be covered by the proposal.
+
+Rationale
+========
+The rationale fleshes out the specification by describing why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other enviroments or scenarios.
+
+Specification
+=============
+The technical specification should describe the aspects of what is being proposed.
+
+Backwards Compatibility
+<<<<<<< HEAD
```

Comment:
Perhaps an error occurred while merging this modification.
`<<<<<<<HEAD` in line 103

Reply:
Thanks. I just fix it.